# Mapping Immune Landscape in Clear Cell Renal Carcinoma by Single-Cell Genomics

Here we provide the processed data and code for single-cell mRNA and T-cell receptor sequencing for immune cells derived from the peripheral blood and tumor-infitlrating poulations in 3 ccRCC patients. The analysis is broken down into four major steps - 1) The integration of single-cell sequencing, 2) CD8+ T cells, 3) CD4+ T cells and 4) Antigen-presenting cells, reflected in the markdown files and folders in the subdirectory *"CellType"*.

## Abstract
**Purpose:** Human clear cell renal cell carcinoma (ccRCC) is one of the most immunologically distinct tumor types due to high response rate to immunotherapies, despite low tumor mutational burden. In contrast to other cancers where higher cytotoxic CD8+ T cells are generally associated with better prognosis, increased CD8+ T cell infiltration is associated with poorer overall survival in ccRCC, suggesting that sub-populations of CD8+ T cells and/or the interaction with other cell types may underlie this observation. 

**Experimental Design:** To characterize the tumor immune microenvironment of ccRCC, we applied single-cell-RNA sequencing (SCRS) along with T-cell-receptor (TCR) sequencing to map the transcriptomic heterogeneity of 25,688 individual CD45+ lymphoid and myeloid cells in matched tumor and blood from patients with ccRCC. We also included 11,367 additional immune cells from normal kidney and peripheral blood to facilitate the identification and assessment of ccRCC-specific differences. 

**Results:** There is an overall increase in intratumoral CD8+ T cell and macrophage populations in tumor-infiltrated immune cells compared to normal renal tissue. We further demonstrate the divergent cell transcriptional states for tumor-infiltrating CD8+ T cells and identify a proliferative subpopulation being a potential culprit for the progression of ccRCC. Tumor-associated macrophages (TAMs) are separated into three distinct clusters, with differences in the expression of angiogenic and secretory genes. 

**Conclusions:** Using the SCRS gene expression, we found preferential prediction of clinical outcomes and pathological diseases by subcluster assignment. With further characterization and functional validation, our findings may reveal certain sub-populations of immune cells amenable to therapeutic intervention.

## Getting Processed Data
Due to the limit of GitHub repo file size, please find a copy of this repository with the processed data at [10.5281/zenodo.4311824](https://zenodo.org/record/4311825).

## Citation
[Article](https://www.nature.com/articles/s42003-020-01625-6): Borcherding, N., Vishwakarma, A., Voigt, A.P. et al. Mapping the immune environment in clear cell renal carcinoma by single-cell genomics. Commun Biol 4, 122 (2021). https://doi.org/10.1038/s42003-020-01625-6. 
